cmake/ctest/drivers/atdm/mutrino/local-driver.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -l
+
+if [ "${SRUN_CTEST_TIME_LIMIT_MINUTES}" == "" ] ; then
+  SRUN_CTEST_TIME_LIMIT_MINUTES=180  # Default limit is 3 hours
+fi
+
+set -x
+
+export CTEST_DO_TEST=OFF
+
+#/usr/local/bin/salloc -N 1 -p standard -J $JOB_NAME \
```

Comment:
@fryeguy52, is this not actually running any tests on mutrino?

Reply:
It is like this because we need to build on the login nodes and test on the compute nodes.  So I would run smart-driver to build and then go into the build directory to run ctest with salloc.  I know you fixed that issue on serrano.  What is the right way to handle it now?